app/bundles/PageBundle/Form/Type/PageListType.php

```diff
@@ -57,21 +60,25 @@ public function configureOptions(OptionsResolver $resolver)
                         $choices[$page['language']][$page['id']] = "{$page['title']} ({$page['id']})";
                     }
 
+                    $choices = array_flip($choices);
+
                     //sort by language
                     ksort($choices);
 
                     foreach ($choices as $lang => &$pages) {
+                        $pages = array_flip($pages);
```

Comment:
If you flip it in the foreach above when the choices array is being built then you won't need this.